.mkrepo/gitattributes.w

```diff
@@ -0,0 +1,16 @@
+bring fs;
+
+pub class GitAttributes {
+  new() {
+    let lines = MutArray<str>[];
+    lines.push("# Auto-generated by ./mkrepo.sh");
+    lines.push("");
+    lines.push("/.github/workflows/*-pull.yaml linguist-generated");
```

Comment:
I would have probably been nicer to invert the control for stuff like this (add a method `addGenerated(glob)` and call it from `Library`.

Reply:
Yeah this was just quick and dirty 😅  it probably make sense to abstract it a bit more eventually